src/data.js

```diff
@@ -242,6 +242,29 @@ module.exports = [
       'CSS',
     ],
   },
+  {
+    name: "Fabian Vallejos",
+    description: "Web Developer, Amateur Photographer, Writer, Occasional Gamer & Streamer, Father of Shih Tzus",
+    url: "https://fabianvallejos.com/uses/",
+    twitter: "@VallejosFab",
+    emoji: "🧉",
+    country: "🇺🇾",
+    computer: "apple",
+    phone: "iphone",
+    tags: [
+      "Web Developer",
+      "Full Stack",
+      "Javascript",
+      "Typescript",
```

Comment:
For "Javascript" and "Typescript" could you capitalise the S's to keep them consistent with other entries? Thanks!